### ISSUE TITLE
I can't remember the last time I got a comment on an open exercise.

### ISSUE THREAD

**Issue by jkrmr** [exercism/exercism.io#2857]
Or felt compelled to leave one for someone else.

Just a head's up in case anyone else has come across this 😉 

Seems like a bug.

**Comment by kytrinyx** [comment]
This is a huge problem in several tracks. It's not the case in all tracks, but even where it's more common to get feedback, there are still not enough people giving feedback.

I've been doing some experiments to see if we can find a way to address it.

**Comment by jkrmr** [comment]
That's reassuring -- I was beginning to wonder if it might just be something wrong with my account.

I vaguely remember arguing (feebly) for required feedback at some point in the distant past, but I see it's been [suggested more recently as well](https://github.com/exercism/exercism.io/issues/2803), so I won't belabor the point. 

I just read about the funding for the site -- that's awesome, congrats! 
I'm looking forward to seeing what's in store 😄 

ref: #2803

**Comment by kytrinyx** [comment]
I still have hopes of figuring out how to solve this without using mechanisms that can be easily gamed with terrible results (e.g. really bad feedback given just so people can move forward).

I did an experiment in the Go track recently which was very promising. I wrote a bit about it in one of the recent newsletters: http://tinyletter.com/exercism/letters/how-do-we-get-people-to-give-more-feedback-on-exercism
